boxoffice/views/admin_report.py

```diff
@@ -169,3 +170,23 @@ def row_handler(row):
         return dict_row
 
     return csv_response(headers, rows, row_type='dict', row_handler=row_handler)
+
+
+@app.route('/admin/o/<org_name>/settlements.csv')
+@lastuser.requires_login
+@load_models(
+    (Organization, {'name': 'org_name'}, 'organization'),
+    permission='org_admin')
+def settled_transactions(organization):
+    if not organization.merchant:
+        abort(401)
+    year = int(request.args.get('year'))
+    month = int(request.args.get('month'))
+    try:
+        date(year, month, 1)
+    except (ValueError, TypeError):
+        return api_error(message='Invalid year/month',
+                status_code=403,
+                errors=['invalid_date'])
+    headers, rows = get_settled_transactions({'year': year, 'month': month}, g.user.timezone)
```

Comment:
This call does not specify the organization it's being called for. Since the RazorPay API key is global to the app, there is no reason to drag the organization into this. Just use Lastuser's `siteadmin` permission with a `@lastuser.requires_permission('siteadmin')` on this view. Or use a new permission name if `siteadmin` is too much to give away.